mail/templates/product_order_receipt/body.html

```diff
@@ -30,6 +30,7 @@ <h3 style="color: #000000; font-size: 16px; font-weight: 700; line-height: 18px;
                 <strong style="font-weight: 700;">Quantity:</strong>  {{ line.quantity }}<br>
                 <strong style="font-weight: 700;">Unit Price:</strong> ${{ line.price }}<br>
                 <strong style="font-weight: 700;">Discount:</strong> ${{ line.discount }}<br>
+                <strong style="font-weight: 700;">Tax:</strong> ${{ line.tax_paid }}<br>
                 <strong style="font-weight: 700;">Total Paid:</strong> ${{ line.total_paid }}
```

Comment:
These need to be quantized - in testing I got a Tax line that was `$21.675000` (which is right, but too many decimals). 

FWIW - I added a couple of template tags in the `mail` app to handle this too; they do work differently than the route you've taken though.

Reply:
Should be fixed in https://github.com/mitodl/mitxpro/pull/2773/commits/426b61ec58c98f6ef73cef9dbd773e5997a43b47